### ISSUE TITLE
Recent music page

### ISSUE THREAD

**Issue by L3XaS** [karimknaebel/phonograph-issue-tracker#144]
Can't create playlist in meizu iPhone running flyme OS

**Comment by arkon** [comment]
Is this still happening?

**Comment by L3XaS** [comment]
Nope. It's running fine. And one request Add a homepage for recent music
like Sony Music 🙏

**Comment by arkon** [comment]
There is a "History" auto playlist which is what you've recently played, but I'll rename this issue for that.
